### PR TITLE
Grabbag of small fixes

### PR DESCRIPTION
**Pull request by svanoort**
* Solves  [JENKINS-33498](https://issues.jenkins-ci.org/browse/JENKINS-33498) by CSS changes + adding class to the fullscreen view main-panel. 
* Solves [JENKINS-33351](https://issues.jenkins-ci.org/browse/JENKINS-33351) by using relativePath when too many artifacts are set to generate displayPaths (API fix).
* Solve [JENKINS-33714](https://issues.jenkins-ci.org/browse/JENKINS-33714) - Correctly calculate the prefixes for data size

### CODE REVIEW

**Comment by svanoort** [comment]
@reviewbybees

**Comment by lordofthejars** [comment]
:bee:

File: ui/src/main/js/view/remove-sidepanel.js [hunk 1]
```diff
@@ -27,4 +27,5 @@ var jqProxy = require('../jQuery');
 exports.render = function (message, onElement) {
     var $ = jqProxy.getJQuery();
     $('div#side-panel').remove();
+    $('div#main-panel').addClass("fullscreen");
```
Comment (recena) [inline ui/src/main/js/view/remove-sidepanel.js]: 🐛  the Jenkins core layout should not be modify in this way.

**Comment by recena** [comment]
🐜  Easier one PR per Jira issue.

File: ui/src/main/js/view/remove-sidepanel.js (hunk 1 above)
Comment (svanoort) [inline ui/src/main/js/view/remove-sidepanel.js]: @recena We're already removing the side-panel in this fashion.  Do you have a better way?  This was the approach that I was able to get working after several attempts, and is quite straight-forward.

**Comment by recena** [comment]
@svanoort You should wait to https://github.com/jenkinsci/jenkins/pull/2253

File: ui/src/main/js/view/remove-sidepanel.js (hunk 1 above)
Comment (recena) [inline ui/src/main/js/view/remove-sidepanel.js]: @svanoort The solution proposes is here https://github.com/jenkinsci/jenkins/pull/2253

**Comment by svanoort** [comment]
@recena I'd be happy to modify this to take that up, once most users are on a core LTS containing that.  However, that will not be true for months at least.  Do you have a proposal that does not require blocking this PR for months?

File: ui/src/main/js/view/remove-sidepanel.js (hunk 1 above)
Comment (recena) [inline ui/src/main/js/view/remove-sidepanel.js]: 🐝  In order to block your PR but when this PR is merged https://github.com/jenkinsci/jenkins/pull/2253 you should review this again.

Comment (svanoort) [inline ui/src/main/js/view/remove-sidepanel.js]: Absolutely - once that capability is in core, it's a much better way of doing things 👍

**Pull request merged**